src/material/schematics/ng-update/upgrade-rules/hammer-gestures-v9/hammer-gestures-rule.ts

```diff
@@ -228,7 +230,8 @@ export class HammerGesturesRule extends MigrationRule<null> {
       this.printInfo(
           'The HammerJS v9 migration for Angular Components migrated the ' +
           'project to keep HammerJS installed, but detected ambiguous usage of HammerJS. Please ' +
-          'manually check if you can remove HammerJS from your application.');
+          'manually check if you can remove HammerJS from your application. More details: ' +
+          'https://git.io/ng-material-v9-hammer-ambiguous-usage');
```

Comment:
This overall message still feels somewhat exaggerated. It's not always ambiguous, and only rarely (see case 1 in the guide). @jelbourn do you have any idea how we can rephrase this and make it not sound like it's always wrong?

Reply:
I think it's fine